deploy/Chart/templates/controller/configmaps.yaml

```diff
@@ -23,7 +23,7 @@ data:
 
     queueProvider:
       provider: "apiserver"
-      name: "ucp"
+      name: "controller"
```

Comment:
🧐

Reply:
This is another thing I want to fix. Due to sharing the configuration with other components, the controller **requires** a configuration for the queue. It doesn't talk to the queue!